Gemfile

```diff
@@ -6,26 +6,20 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 # development dependencies will be added by default to the :development group.
 gemspec
 
+gem 'pagy', '~> 3.8'
```

Comment:
If it's on the `gemspec`, does it also need to be here?

Reply:
i thought i had this deleted🤔